atomic_reactor/koji_util.py

```diff
@@ -166,12 +166,14 @@ def tag_koji_build(session, build_id, target, poll_interval=5):
     return build_tag
 
 
-def get_koji_task_owner(session, task_id, default=""):
+def get_koji_task_owner(session, task_id, default=None):
+    default = {} if default is None else default
     if task_id:
         try:
             koji_task_info = session.getTaskInfo(task_id)
             koji_task_owner = session.getUser(koji_task_info['owner'])
-        except:
+        except Exception as e:
+            logger.warn('Exception getting task owner: {} {}'.format(type(e), e))
```

Comment:
Thanks for adding this log message. Consider using `logger.exception` instead:
```python
logger.exception('Unable to get Koji task owner')
```

^ That will actually log the whole traceback as context and provide the useful error message to users.

Reply:
Sure. I didn't have the context as to what kinds of things might cause this to fail and how much anything would care, I just knew it was confusing when I had a bug in my unit test